tests/yam/agama/agama_bundle.pm

```diff
@@ -0,0 +1,27 @@
+## Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run interactive installation with Agama,
+# using a web automation tool to test directly from the Live ISO.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::agama_base;
+use strict;
+use warnings;
+use testapi qw(assert_script_run get_required_var get_var script_run select_console);
+
+sub run {
+    my $self = shift;
+    my $test = get_required_var('AGAMA_TEST');
+    my $delay = get_var('AGAMA_DELAY', 0);
+    my $reboot_page = $testapi::distri->get_reboot_page();
+
+    script_run("dmesg --console-off");
+    assert_script_run("node --test-name-pattern='$test' /usr/share/agama/system-tests/bundle.cjs --delay $delay");
```

Comment:
please, remove delay, I don't want to encourage this, I'm really seeing the future here, someone adding a delay and nobody noticing for long time. Let's hack the script in our branch with this when needed.
Other solution could be to have a ci check to prevent in job group to add the setting in production, but we are not there, file a ticket if you want to not forget, I understand that it can be comfortable to have to debug.